cms/djangoapps/modulestore_migrator/api/write_api.py

```diff
@@ -0,0 +1,94 @@
+"""
+API for kicking off new migrations
+"""
+from __future__ import annotations
+
+from celery.result import AsyncResult
+from opaque_keys.edx.locator import LibraryLocatorV2
+from openedx_learning.api.authoring import get_collection
+
+from openedx.core.types.user import AuthUser
+from openedx.core.djangoapps.content_libraries.api import get_library
+
+from ..data import SourceContextKey
+from .. import tasks, models
+
+
+__all__ = (
+    'start_migration_to_library',
+    'start_bulk_migration_to_library'
+)
+
+
+def start_migration_to_library(
+    *,
+    user: AuthUser,
+    source_key: SourceContextKey,
+    target_library_key: LibraryLocatorV2,
+    target_collection_slug: str | None = None,
+    create_collection: bool = False,
+    composition_level: str,
+    repeat_handling_strategy: str,
+    preserve_url_slugs: bool,
+    forward_source_to_target: bool | None
+) -> AsyncResult:
+    """
+    Import a course or legacy library into a V2 library (or, a collection within a V2 library).
+    """
+    return start_bulk_migration_to_library(
+        user=user,
+        source_key_list=[source_key],
+        target_library_key=target_library_key,
+        target_collection_slug_list=[target_collection_slug],
+        create_collections=create_collection,
+        composition_level=composition_level,
+        repeat_handling_strategy=repeat_handling_strategy,
+        preserve_url_slugs=preserve_url_slugs,
+        forward_source_to_target=forward_source_to_target,
+    )
+
+
+def start_bulk_migration_to_library(
```

Comment:
This api method is virtually unchanged. Its definition has just been moved from `modulestore_migrator.api` down into `modulestore_migrator.api.write_api`.